File: acme-java-springboot/src/main/java/acme/repository/CustomerAccountRepository.java
package acme.repository;

import acme.entity.CustomerAccount;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

import java.util.List;
import java.util.UUID;

@Repository
public interface CustomerAccountRepository extends CrudRepository<CustomerAccount, UUID> {
    List<CustomerAccount> findByCustomerId(UUID customerId);
}

File: acme-springboot/src/main/kotlin/acme/repository/CustomerAccountRepository.kt
package acme.repository

import acme.entity.CustomerAccount
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository
import java.util.*

@Repository
interface CustomerAccountRepository : CrudRepository<CustomerAccount, UUID> {
    fun findByCustomerId(customerId: UUID): List<CustomerAccount>
}
